doc/HOWTO_RELEASE.rst.txt

```diff
@@ -40,27 +40,27 @@ Release Scripts
 
 Supported platforms and versions
 ================================
-Python 2.7 and >=3.4 are the currently supported versions when building from
-source.  We test NumPy against all these versions every time we merge code to
-master.  Binary installers may be available for a subset of these versions (see
-below).
+`NEP 29`<https://numpy.org/neps/nep-0029-deprecation_policy.html>_ outlines
+which Python versions are supported; For the first half of 2020, this will be
+Python >= 3.6. We test NumPy against all these versions every time we merge
+code to master.  Binary installers may be available for a subset of these
+versions (see below).
 
 
 OS X
 ----
-Python 2.7 and >=3.4 are the versions for which we provide binary installers.
-OS X versions >= 10.6 are supported.  We build binary wheels for OSX that are
-compatible with Python.org Python, system Python, homebrew and macports - see
-this `OSX wheel building summary
+OS X versions >= 10.6 are supported, for Python version support see NEP 29.
+We build binary wheels for OSX that are compatible with Python.org Python,
+system Python, homebrew and macports - see this `OSX wheel building summary
 <https://github.com/MacPython/wiki/wiki/Spinning-wheels>`_ for details.
 
 
 Windows
 -------
-We build 32- and 64-bit wheels for Python 2.7, 3.4, 3.5 on Windows. Windows
-XP, Vista, 7, 8 and 10 are supported.  We build NumPy using the MSVC compilers
-on Appveyor, but we are hoping to update to a `mingw-w64 toolchain
-<https://mingwpy.github.io>`_.  The Windows wheels use ATLAS for BLAS / LAPACK.
+We build 32- and 64-bit wheels on Windows. Windows XP, Vista, 7, 8 and 10 are
```

Comment:
I didn't find any reference to windows versions in travis or appveyor, where can I double check XP is still supported?

Reply:
XP hasn't been supported for quite a while. Less sure about Windows 7, I think that may be the lowest version. Here's the wheel build repo: https://github.com/MacPython/numpy-wheels

Reply:
It looks like windows ended support for Vista in [April 2017](https://support.microsoft.com/en-us/help/22882/windows-vista-end-of-support) so I'm going to remove it from the list.